src/test/java/edu/kis/powp/jobs2d/TestJobs2dApp.java

```diff
@@ -33,8 +35,29 @@ private static void setupPresetTests(Application application) {
 		SelectTestFigure2OptionListener selectTestFigure2OptionListener = new SelectTestFigure2OptionListener(
 				DriverFeature.getDriverManager());
 
+
+		ActionListener rectangle = new ActionListener()
+		{
+			public void actionPerformed(ActionEvent e) {
+				ComplexCommandFactory.drawRectangle(0,0, 50, 25);
+			}
+		};
+
+		ActionListener triangle = new ActionListener()
+		{
+			public void actionPerformed(ActionEvent e)
+			{
+				ComplexCommandFactory.drawTriangle(-50,-50, 50, 50);
+			}
+		};
+
```

Comment:
It is good to keep a convention used in the original code (class instance instead of anonymous classes). It pollutes the code too much.